### DISCUSSION TITLE
Welcome to Catalyst-Circle-Co-ordination Discussions!

### DISCUSSION THREAD

**Discussion by stephen-rowan** [Catalyst-Swarm/-Circle-of-Toolmakers-and-Maintainers#58] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by xeeban** [comment]
Thanks for setting this up, Stephen! Should we assign some basic tasks to people to ensure they are able to log in, get around, and use this tool? I am sure it will be new to most.

**Comment by xeeban** [comment]
Oh, I guess I should follow instructions and introduce myself and what I do around here. I am Nori Nishigaya. I am the founder of the [Salmon Nation Decentralized Alliance](https://www.sanada.eco), a Cardano local community hub, and an active participant of Catalyst Town Halls, Catalyst Swarm, CC General Voter selection committee, member of the CC Admin team, an SPO co-running Salmon Pool, and a Catalyst Fund 6 CA and proposer, and a member of several challenge team proposals (Developer Ecosystem, Rapid Funding Mechanisms, Climate Change: THE Challenge, and Increasing Diversity in Catalyst). Whew, I've never written all that out before... seems like I have definitely taken the red pill.
